packages/fixie-hello-world/package.json

```diff
@@ -2,55 +2,19 @@
   "name": "fixie-hello-world",
   "version": "1.0.0",
   "devDependencies": {
+    "@fixieai/sdk": "workspace:*",
     "@tsconfig/node18": "^2.0.1",
-    "@types/lodash": "^4.14.195",
-    "@types/node": "^20.3.1",
-    "@types/prompt-sync": "^4.2.0",
-    "@types/uuid": "^9.0.2",
-    "@types/yargs": "^17.0.24",
-    "@typescript-eslint/eslint-plugin": "^5.60.0",
-    "@typescript-eslint/parser": "^5.60.0",
-    "eslint": "^8.40.0",
-    "eslint-config-nth": "^2.0.1",
     "typescript": "^5.1.3"
   },
   "private": true,
   "type": "module",
   "scripts": {
-    "dev": "yarn build && npx ts-node dist/serve-bin.js --port 8080 --package-path fixie",
-    "lint": "eslint . --max-warnings 0",
-    "lint:fix": "eslint . --fix",
```

Comment:
Why not lint this package?

Reply:
The linter config/machinery/dependencies were significantly larger than the meaningful contents of the package so I removed them to try to maximize the signal to noise ratio. Was that a mistake?

Reply:
I like to lint all day erry day but it's not a big deal. For having 10 lines of real code it's probably fine to skip; TS gives us basic protection anyway.